### ISSUE TITLE
Horizontal layout on local changes

### ISSUE THREAD

**Issue by LouisPhilippeHeon** [ForkIssues/TrackerWin#2175]
I made a rough mockup of what I would like the choice to have (and what is already there, for comparison).
![Layout Classique](https://github.com/fork-dev/TrackerWin/assets/83369199/e7678d70-f91d-46c7-8189-11178778f40c)
![Nouveau Layout](https://github.com/fork-dev/TrackerWin/assets/83369199/a76950d0-2438-4091-9f9f-9b9e3c4e0eda)
The setting could be added in a new section of the appearance menu called "Local changes layout"

The benefit of having such a layout would be for people with vertical monitors, small laptop screens or in the case the window occupies half the screen.
Another benefit would be making more space for side by side view, even people with big screens could see a benefit from this.

**Comment by DanPristupov** [comment]
> Another benefit would be making more space for side by side view, even people with big screens could see a benefit from this.

Yes, but an element that is at the bottom cannot change an element that is at the top. The traditional Western flow from left to right, from top to bottom should not be broken.

So most likely such a layout will not appear in the near future.

**Comment by LouisPhilippeHeon** [comment]
It could be the other way around, all I want is a practical layout for my vertical monitor.

**Comment by LouisPhilippeHeon** [comment]
This would respect the traditionnal western flow
![Nouveau Layout Inversé](https://github.com/fork-dev/TrackerWin/assets/83369199/19d9d706-efd1-4bc2-aee0-d6a055e9458a)
